packages/editor/src/components/block-vertical-alignment-toolbar/test/index.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+import { shallow } from 'enzyme';
+
+/**
+ * Internal dependencies
+ */
+import { BlockVerticalAlignmentToolbar } from '../';
+
+describe( 'BlockVerticalAlignmentToolbar', () => {
+	const alignment = 'top';
+	const onChange = jest.fn();
+
+	const wrapper = shallow( <BlockVerticalAlignmentToolbar value={ alignment } onChange={ onChange } /> );
+
+	const controls = wrapper.props().controls;
+
+	afterEach( () => {
+		onChange.mockClear();
+	} );
+
+	it( 'should match snapshot', () => {
+		expect( wrapper ).toMatchSnapshot();
+	} );
+
+	it( 'should call onChange with undefined, when the control is already active', () => {
+		const activeControl = controls.find( ( { title } ) => title.toLowerCase().includes( alignment ) );
+		activeControl.onClick();
+
+		expect( activeControl.isActive ).toBe( true );
+		expect( onChange ).toHaveBeenCalledTimes( 1 );
+		expect( onChange ).toHaveBeenCalledWith( undefined );
+	} );
+
+	it( 'should call onChange with alignment value when the control is inactive', () => {
+		const inactiveCenterControl = controls.find( ( { title } ) => title.toLowerCase().includes( 'middle' ) ); // note "middle" alias for "center"
```

Comment:
another nitpick, but I think the coding style mentions that comments should be above the line rather than inline:
https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#comments

Reply:
Fixed https://github.com/WordPress/gutenberg/pull/13899/commits/5c58ca4d3614b185caeabe5bd5c0f2f96abb4e79